source/georef/tests/georef_test.cpp

```diff
@@ -398,19 +398,21 @@ BOOST_AUTO_TEST_CASE(compute_nearest){
 
     w.init(starting_point);//not mandatory, but reinit to clean the distance table to get fresh dijsktra
     res = w.find_nearest_stop_points(100_s, pl, false);
-    BOOST_REQUIRE_EQUAL(res.size(), 1);
-    BOOST_CHECK_EQUAL(res[0].first , 0);
-    BOOST_CHECK_EQUAL(res[0].second, navitia::seconds(60 / (default_speed[Mode_e::Walking] * 2))); //the projection is done with the same mean of transport, at the same speed
+    //the projection is done with the same mean of transport, at the same speed
+    navitia::routing::map_stop_point_duration tested_map;
+    tested_map.insert(navitia::routing::stop_point_duration(navitia::routing::SpIdx(0),
```

Comment:
I think it's clearer if you build the SpIdx directly with the stop point

`navitia::routing::SpIdx(sp1)`